openlibrary/macros/ReadingLogButton.html

```diff
@@ -1,13 +1,26 @@
-$def with (work, read_status)
+$def with (work, read_status, url=None, searchPage = None)
```

Comment:
```suggestion
$def with (work, read_status, url=None, form_classes="")
```
The more generic `form_classes` will allow us to easily add new classes in the future.